### PR TITLE
Fix slide question content locked

### PR DESCRIPTION
**Pull request by antonia2929**
This PR solved this issue : 
https://trello.com/c/4drmAgoB/2718-componentsapp-review-bloquer-options-click-on-slide-after-validate-question

**Detailed purpose of the PR**
When correction poppin is enabled, the question content is unlocked.

**Result and observation**
### BEFORE
Kapture 2022-09-16 at 17 21 44

### AFTER
Kapture 2022-09-16 at 17 23 20

**Testing Strategy**
- Already covered by tests
- Manual testing

### CODE REVIEW

File: packages/@coorpacademy-components/src/organism/review-slide/index.js [hunk 1]
```diff
@@ -137,7 +144,8 @@ const QuestionContainer = props => {
 QuestionContainer.propTypes = {
   answerUI: PropTypes.shape(propTypes.slide.answerUI),
   questionText: PropTypes.string,
-  questionOrigin: PropTypes.string
+  questionOrigin: PropTypes.string,
+  disabledContent: PropTypes.bool
```
Comment (StefanoA1) [inline packages/@coorpacademy-components/src/organism/review-slide/index.js:148]: Juste du wording
```suggestion
  disableContent: PropTypes.bool
```

File: packages/@coorpacademy-components/src/organism/review-slide/style.css [hunk 2]
```diff
@@ -138,6 +138,10 @@ _:-ms-fullscreen, :root .correctionPopinWrapper {
   width: 60px;
 }
 
+.disableSlideContent {
```
Comment (StefanoA1) [inline packages/@coorpacademy-components/src/organism/review-slide/style.css:141]:
```suggestion
.disabledSlideContent {
```

File: packages/@coorpacademy-components/src/organism/review-slide/index.js [hunk 3]
```diff
@@ -114,7 +114,14 @@ const QuestionContainer = props => {
     : answerUI;
 
   return (
-    <div key="content-container" className={style.slideContentContainer}>
+    <div
+      key="content-container"
+      className={
+        disabledContent
+          ? style.slideContentContainer
+          : classnames(style.slideContentContainer, style.disableSlideContent)
```
Comment (StefanoA1) [inline packages/@coorpacademy-components/src/organism/review-slide/index.js:122]:
```suggestion
          : classnames(style.slideContentContainer, style.disabledSlideContent)
```

**Review by StefanoA1 — CHANGES_REQUESTED** [review]

File: packages/@coorpacademy-components/src/organism/review-slide/index.js [hunk 4]
```diff
@@ -114,7 +114,14 @@ const QuestionContainer = props => {
     : answerUI;
 
   return (
-    <div key="content-container" className={style.slideContentContainer}>
+    <div
+      key="content-container"
+      className={
+        disabledContent
```
Comment (jomaora) [inline packages/@coorpacademy-components/src/organism/review-slide/index.js:120]:
```js
classnames(
   style.slideContentContainer,
   disabledContent ? style.disableSlideContent : null
)
```

**Review by jomaora — CHANGES_REQUESTED** [review]

**Review by esa-coorp — CHANGES_REQUESTED** [review]

File: packages/@coorpacademy-components/src/organism/review-slide/index.js [hunk 5]
```diff
@@ -165,6 +173,7 @@ const ReviewSlide = props => {
             questionOrigin={parentContentTitle}
             questionText={questionText}
             answerUI={answerUI}
+            disabled={!showCorrectionPopin}
```
Comment (jomaora) [inline packages/@coorpacademy-components/src/organism/review-slide/index.js:176]: ici tu utilises `disabled` mais dans les props t'as `disableContent`... du coup, je suis surpris que ça marche

File: packages/@coorpacademy-components/src/organism/review-slide/index.js [hunk 6]
```diff
@@ -137,7 +144,8 @@ const QuestionContainer = props => {
 QuestionContainer.propTypes = {
   answerUI: PropTypes.shape(propTypes.slide.answerUI),
   questionText: PropTypes.string,
-  questionOrigin: PropTypes.string
+  questionOrigin: PropTypes.string,
+  disableContent: PropTypes.bool
```
Comment (jomaora) [inline packages/@coorpacademy-components/src/organism/review-slide/index.js:148]: `disabledContent` ou `disable`

**Review by jomaora — CHANGES_REQUESTED** [review]

**Review by jomaora — APPROVED** [review]

**Review by jomaora — APPROVED** [review]

**Review by StefanoA1 — APPROVED** [review]

**Pull request merged**